### PR TITLE
Excel Converter Prototype 1

### PR DESCRIPTION
**Pull request by ryan-c-2024**
This PR highlights my Excel converter code and compares it to the original PowerFX repo (and the state it was in as I initially forked it). I'd appreciate any feedback!

### CODE REVIEW

File: src/Directory.Build.props [hunk 1]
```diff
@@ -94,4 +94,4 @@
   <PropertyGroup>
     <DefaultItemExcludes>.git\**;**\node_modules\**;$(DefaultItemExcludes)</DefaultItemExcludes>
   </PropertyGroup>
-</Project>
```
Comment (jengregory) [inline src/Directory.Build.props:97]: testing

File: src/libraries/ExcelConverter/ExcelParser.cs [hunk 2]
```diff
@@ -0,0 +1,420 @@
...
+            public string CellId;
+            public string Formula;
+            public string Value;
+        }
+
+     
+            public static ParsedExcelData ParseSpreadsheet(String excelFilePath, bool outputFile = true)
+            {
+                TestCellId();
+                TestSharedFormulaResolve();
```
Comment (jengregory) [inline src/libraries/ExcelConverter/ExcelParser.cs:73]: I think you can remove these from your code - this looks like something that was in the original parser just to test things, but it's not really needed for parsing the workbook

`TestCellId();
TestSharedFormulaResolve();`

File: src/libraries/ExcelConverter/ParsedCellAnalyzer.cs [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+using System.Net.Http.Headers;
+using System.Text;
+using Microsoft.PowerFx;
+using Microsoft.PowerFx.Core;
+using Microsoft.PowerFx.Syntax;
+
+
+namespace ExcelConverter
+{
+    /*
```
Comment (jengregory) [inline src/libraries/ExcelConverter/ParsedCellAnalyzer.cs:15]: Is this still the same flow? If not, you can update it to reflect what's happening here

File: src/libraries/ExcelConverter/ParsedCellAnalyzer.cs [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+        }
+
+        public override String Visit(BinaryOpNode node, Precedence context)
+        {
+            var binaryOpNode = (BinaryOpNode)node;
+
+            String left = node.Left.Accept(this, Precedence.None);
+            String right = node.Right.Accept(this, Precedence.None);
+
+            return left + " + " + right;
```
Comment (jengregory) [inline src/libraries/ExcelConverter/ParsedCellAnalyzer.cs:113]: should + be hard coded? Or should it be the Op value?

File: src/libraries/Microsoft.PowerFx.Core/Parser/TexlParser.cs [hunk 5]
```diff
@@ -181,7 +181,7 @@ private TexlNode Parse(ref List<TexlError> errors)
                 PostBlockCommentMissingClosingError();
 
                 errors = _errors;
-            }
```
Comment (jengregory) [inline src/libraries/Microsoft.PowerFx.Core/Parser/TexlParser.cs:184]: if this is just a whitespace change, you can revert the changes in this file

File: src/libraries/ExcelConverter/ParsedCellAnalyzer.cs (hunk 4 above)
Comment (ryan-c-2024) [reply src/libraries/ExcelConverter/ParsedCellAnalyzer.cs:113]: Fixed this bug earlier - good catch!
